src/main/java/org/cactoos/text/CapitalizedText.java

```diff
@@ -53,15 +53,15 @@ public CapitalizedText(final Text text) {
     @Override
     public String asString() throws IOException {
         return new JoinedText(
-            new TransformedIterable<String, Text>(
+            new TransformedIterable<Text, Text>(
                 new SplitText(this.origin),
-                new Func<String, Text>() {
+                new Func<Text, Text>() {
                     @Override
-                    public Text apply(final String input) throws Exception {
+                    public Text apply(final Text input) throws Exception {
```

Comment:
@Timmeey Well, I think private classes or methods hide functionality. So, we could improve reusability making all (less attributes) public. Today, I can see no funcionality for these classes, but in the future? If there is no funcionality, the user won't use them. What do you think?